documentation/modules/ref-message-redelivery.adoc

```diff
@@ -0,0 +1,5 @@
+
+The `messageRedelivery` field is used to control the behavior of a queue or subscription when a receiver is unsuccessful in its processing
+of a message and the mesage is returned to the queue for redelivery.   By default, messages will be redelivered indefinitely.
```

Comment:
@hmaclean73 the sentence that follows need to change to parse with your change.
```suggestion
a message and the mesage is returned to the queue for redelivery.   By default, messages will be redelivered indefinitely.
```

Reply:
LGTM